spec/factories/sequences.rb

```diff
@@ -8,4 +8,6 @@ def random_grid_reference
     last = ("A".."Z").to_a.reject { |a| a == "I" }.sample
     [first, last].join
   end
+
+  sequence(:random_email) { Faker::Internet.email }
```

Comment:
I'd suggest using `Faker::Internet.safe_email` here - just in case something does wrong and the test system starts sending emails out.